src/view/widgets/manga.rs

```diff
@@ -37,28 +37,35 @@ impl Widget for ChapterItem {
         Self: Sized,
     {
         let layout = Layout::horizontal([
-            Constraint::Percentage(50),
-            Constraint::Percentage(30),
-            Constraint::Percentage(20),
+            Constraint::Length(3),
+            Constraint::Length(3),
+            Constraint::Length(3),
+            Constraint::Fill(50),
+            Constraint::Fill(30),
+            Constraint::Fill(20),
         ]);
 
-        Block::bordered().border_style(self.style).render(area, buf);
-
-        let [title_area, scanlator_area, readable_at_area] = layout.areas(area.inner(Margin {
-            horizontal: 1,
-            vertical: 1,
-        }));
+        let [is_read_area, is_downloaded_area, lang_area, title_area, scanlator_area, readable_at_area] =
+            layout.areas(area.inner(Margin {
+                horizontal: 0,
+                vertical: 0,
+            }));
```

Comment:
```suggestion
        let [is_read_area, is_downloaded_area, lang_area, title_area, scanlator_area, readable_at_area] = layout.areas(area);
```
Margin { horizontal : 0, vertical : 0 } is redundant

Reply:
done.